src/App.js

```diff
@@ -1,19 +1,37 @@
-import { BottomNav, Navbar, SideNav } from "components";
+import { Navbar, Toast } from "components";
 import { routes } from "config/routes";
+import jwtDecode from "jwt-decode";
+import { useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
 import { useRoutes } from "react-router-dom";
+import { loginFromLocal } from "redux/features/userSlice";
 
 const App = () => {
   const routesEl = useRoutes(routes);
+  const dispatch = useDispatch();
+  const user = useSelector((state) => state.user.user);
```

Comment:
suggestion: you can export this via some custom hook from slice itself ie `export const useAuth = () =>  useSelector(state => state.user)`